install/infra/single-cluster/aws/terraform.tfvars

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
**Question:** If a user tries to use this module, will the presence of this `terraform.tfvars` file conflict with their environment?

Note that this question and any associated changes may be out of scope for this PR, as we're mainly focused on synchronizing the reference architecture and Terraform module. If it is then resolve this issue, we can deal with issues like this in a later PR.

Reply:
Excellent point! I will leave those fields empty, so they will have to edit it in any case